fancy_gym/envs/mujoco/air_hockey/air_hockey_env_wrapper.py

```diff
@@ -39,6 +39,18 @@ def __init__(self, env_mode=None, interpolation_order=3, render_mode=None, **kwa
         if env_mode == "tournament" and type(interpolation_order) != tuple:
             interpolation_order = (interpolation_order, interpolation_order)
 
+        self.render_mode = render_mode
+
+        # Determine headless mode based on render_mode
+        headless = self.render_mode == 'rgb_array'
+        width = kwargs.pop('width', 1920)
+        height = kwargs.pop('height', 1080)
```

Comment:
Is there any reason as to why this is implemented in this fashion?
Could `width` and `height` not be added as regular arguments to the `__init__` function?

How about (prefered)
```
def __init__(self, [...], width=1920, height=1080, viewer_params={})
```
or
```
def __init__(self, [...], viewer_params={'width': 1920, 'height': 1080})
```
?

This approach would offer better IDE support for parameter hinting and automatic documentation generation, such as on ReadTheDocs. Additionally, it would make the code more readable and easier to understand for new developers.

Reply:
You are right, this is unnecessary.